### ISSUE TITLE
"lock" mixin (name TBC)

### ISSUE THREAD

**Issue by alfie-g** [yoyodesign/gort#8]
Mixin for having precise control over a responsive property value.

Inspired by Mike Riethmuller's [repsonsive typography](https://madebymike.com.au/writing/precise-control-responsive-typography/) using `calc()` and `vw`.

The name for this mixin is still to be confirmed, if you ever a better name for it then suggest it below.

```
// Mixin signatures:
@mixin lock($property, $min, $max, $from, $to) {
    ...
}

// Mixin usage:
h1 {
    lock("font-size", 30px, 45px, "breakpoint-1", "breakpoint-2");
}

**Comment by calvindavis** [comment]
Currently you're passing string values for the `$from` and `$to` breakpoints – would this need to tie in to our responsive breakpoint solution?

**Comment by alfie-g** [comment]
Yes that is the idea, but this is also up for discussion. I think the ideal solution would be for these values to either take in a pixel value, or a string value from our breakpoint solution.

**Comment by calvindavis** [comment]
We should probably park this until we've decided on a breakpoints solution – #9.
